### ISSUE TITLE
Lokales Echtzeit-Wetter

### ISSUE THREAD

**Issue by juliahebestreit** [Klima-Dashboard/node-red#3]
- Feinstaub-Sensor Belzig
- https://sensor.community/de/
- Aktuelle Temperatur und andereverfügbare Werte
- Aktualisierung: automatisch, Echtzeit
- Kategorie: Wetter
- Quellen: 12  SensorCommunity und OpenSenseBox

**Comment by juliahebestreit** [comment]
Zwei Sensoren im relevanten Gebiet. Beide in Bad Belzig. 
#62992 und #58258

**Comment by tpwd-moradnia** [comment]
test-dashbord in grafana eingebaut

**Comment by pwilimzig** [comment]
@tpwd-moradnia Bitte mal hier einen Link zu dem Dashboard posten

**Comment by EricMuelling** [comment]
Finde die Feinstaub-Sensoren sehr wichtig für das Dashboard. Sollte rein. Bad Belzig ist schließlich eine Kurstadt und sehr stolz auf seine saubere Luft.

**Comment by pwilimzig** [comment]
@tpwd-moradnia Bitte wieder neu einbauen.

**Comment by tpwd-moradnia** [comment]
#58258 existiert nicht ich glaube die richtige id ist 58256

**Comment by tpwd-moradnia** [comment]
https://grafana.bad-belzig.klima-daten.de/d/h2x-HdoVk/community-daten?orgId=1

**Comment by Wurmdaniel** [comment]
https://data.opensensorweb.de/#/search?c=13.695%2C51.067&te=2021-12-07T09%3A17%3A43.448Z%2C2022-12-07T09%3A17%3A43.448Z&tz=B&v=main&z=11.2
sind jetzt mehr daten online, hat Zeit bis nach dem 1. März!

**Comment by juliahebestreit** [comment]
Hallo @Wurmdaniel , danke für den Link. Bislang haben wir Daten von den Sensoren 58256 und 62992.
Siehe https://grafana.bad-belzig.klima-daten.de/d/h2x-HdoVk/community-daten?orgId=1 
Welche Sensoren möchtest du noch dabei haben? Bzw. wie ist dieser Link zu verstehen?

**Comment by Wurmdaniel** [comment]
@juliahebestreit @pwilimzig  in der region sind derzeit 7 sensoren eingebunden, drarunter auch alle daten der DWD wetterstation, sowie die Luftgüte daten aus Lütte (#23), leider sind nicht alle Daten aktuell (z.B. Coconut). 
Alle aktuellen Daten sollten eingebettet werden!! Hier werden laufend neue Daten eingestellt, deshalb braucht es auch eine niedrigschwellige Anleitung von euch, wie neue Sensoren eingebettet werden können.

**Comment by juliahebestreit** [comment]
Für die Startseite https://bad-belzig.klima-daten.de: Daten von welchem Sensor sollen dargestellt werden? Aktuell werden Daten von zwei Standorten importiert: https://grafana.bad-belzig.klima-daten.de/d/h2x-HdoVk/community-daten?orgId=1 
Soll die Grafik Feinstaub PM10 oder Feinstaub PM 2.5 dargestellt werden? @Wurmdaniel @LisannBach

**Comment by donni106** [comment]
Der zweite Sensor liefert irgendwie keine Daten mehr: https://data.sensor.community/airrohr/v1/sensor/62992/
Ist das bewusst so und richtig?

**Comment by LisannBach** [comment]
> Für die Startseite https://bad-belzig.klima-daten.de: Daten von welchem Sensor sollen dargestellt werden? Aktuell werden Daten von zwei Standorten importiert: https://grafana.bad-belzig.klima-daten.de/d/h2x-HdoVk/community-daten?orgId=1 Soll die Grafik Feinstaub PM10 oder Feinstaub PM 2.5 dargestellt werden? @Wurmdaniel @LisannBach

Bitte beide Werte einbinden. Beide sollten aber unter der gleichen Überschrift "Feinstaub" laufen.
Info Icon Text: Feinstaubwerte können in zwei Kategorien unterteilt werden:  PM10 (PM, particulate matter) mit einem maximalen Durchmesser von 10 Mikrometer (µm), PM2,5 und ultrafeine Partikel mit einem Durchmesser von weniger als 0,1 µm. Der Tagesgrenzwert für PM10 beträgt 50 µg/m3 und darf nicht öfter als 35mal im Jahr überschritten werden. Für PM2,5 beträgt der Grenzwert 25 µg/m3 pro Kalenderjahr.

**Comment by LisannBach** [comment]
> Der zweite Sensor liefert irgendwie keine Daten mehr: https://data.sensor.community/airrohr/v1/sensor/62992/ Ist das bewusst so und richtig?

@Wurmdaniel hast du dazu Infos?

**Comment by donni106** [comment]
Dann ist das widget glaube ich bereits fertig, oder?

![image](https://github.com/Klima-Dashboard/data-tools/assets/1942953/7b397cbf-10b8-4f4e-997a-f29015824a43)

https://grafana.bad-belzig.klima-daten.de/d/h2x-HdoVk/community-daten?orgId=1

**Comment by LisannBach** [comment]
Genau. PM10 und PM2,5. Die Frage die sich mir stellt. Was sollen die Gauge Darstellungen aussagen bzw. die Threshholds? Repräsentieren die die oben genannten Grenzwerte? Dem Betrachter würde nicht klar sein, was die Fülle der Gauge bedeutet. 
Ich fände hier eine einfache Zahl sinnvoll, ohne Gauge. Grün wäre für mich "alles im normalen Bereich" und rot müsste sich einstellen sobald der Grenzwert überschritten ist

**Comment by donni106** [comment]
Dann so:

![image](https://github.com/Klima-Dashboard/data-tools/assets/1942953/4ba8c522-e69e-4e31-b264-1c3c5d77c557)

Grenzwerte habe ich aus deinem Kommentar entnommen und 50 bzw. 25 hinterlegt, ab wann die Werte rot werden statt grün.

**Comment by LisannBach** [comment]
Perfekt! Super. Den Text bitte dann hinter so ein Info i legen in Grafana.

**Comment by juliahebestreit** [comment]
Geht das? 

> Perfekt! Super. Den Text bitte dann hinter so ein Info i legen in Grafana.

 @donni106 
+ welchen Text exakt Lisann?

**Comment by donni106** [comment]
Ja, in "Description" eines Panels: 

![image](https://github.com/Klima-Dashboard/data-tools/assets/1942953/f97b2880-d446-4784-a1b1-1c84413f493d)

**Comment by juliahebestreit** [comment]
Grafik nun angepasst, dass beide Werte in einem Modul sind: https://grafana.bad-belzig.klima-daten.de/d/h2x-HdoVk/community-daten?orgId=1

**Comment by LisannBach** [comment]
Ich hab die Texte in die Description eingefügt. Sonst stehen die weiter oben in der Discussion in diesem Modul

**Comment by juliahebestreit** [comment]
Gibt es eine spezifische Quelle für die beiden Feinstab Werte? Oder ist es diese allgemeine? https://sensor.community/de/ @donni106

**Comment by donni106** [comment]
ja die beiden hier:
https://data.sensor.community/airrohr/v1/sensor/58256/
https://data.sensor.community/airrohr/v1/sensor/62992/

wobei der zweite wie zuletzt mal erwähnt nichts (mehr?) liefert: https://github.com/Klima-Dashboard/data-tools/issues/3#issuecomment-1562578209

> Der zweite Sensor liefert irgendwie keine Daten mehr: https://data.sensor.community/airrohr/v1/sensor/62992/
Ist das bewusst so und richtig?

**Comment by juliahebestreit** [comment]
@LisannBach sollen diese beiden als Quellen benannt werden (siehe Daniels Kommentar) oder die allgemeine von mir benannte url?

**Comment by Wurmdaniel** [comment]
@pwilimzig @donni106 hier die alternative zur sensor.community: https://data.opensensorweb.de

**Comment by juliahebestreit** [comment]
https://github.com/Klima-Dashboard/data-tools/issues/3#issuecomment-1486282064 Kam heute nochmal zur Sprache. 
Wo genau sollen "alle Sensoren" dargestellt werden und wie? Aktuell gibt es die als Teaser auf der Startseite m.E. @Wurmdaniel

**Comment by juliahebestreit** [comment]
Neue Grafik: https://grafana.bad-belzig.klima-daten.de/d/_7-okRX4z/sensor-community

Unter https://grafana.bad-belzig.klima-daten.de/d/_7-okRX4z/sensor-community?orgId=1 ist eine Karte mit allen Sensoren aus der Sensor.Community innerhalb eines größeren Rechtecks um Bad Belzig. Sollten in der Sensor.Community Seite nun weitere Sensoren in diesem Bereich auftauchen, würden diese hier vollautoamatisch erscheinen! Derzeit lassen sich zu den einzelnen Sensoren in der Karte noch keine Detailinfos abrufen, aber das bekommen wir auch noch hin.

Die Ansicht unter https://sensor.community/de/ ist etwas komisch, weil hier zunächst NUR Sensoren angezeigt werden, die Feinstaubwerte liefern. Man muss dann manuell andere Werte wie z.B. “Temperatur” auswählen, um mehr Sensoren zu finden. In Summe werden dann unter also unter https://data.sensor.community/airrohr/v1/filter/box=52.371483,11.518359,51.954793525119456,13.028238446352097 sicher alle Sensoren geliefert, die auch unter https://data.opensensorweb.de/#/search?c=12.59272%2C52.14184&te=2022-06-14T16%3A34%3A25.302Z%2C2023-06-14T16%3A34%3A25.302Z&tz=B&v=main&z=12 angezeigt werden (da OpenSensorWeb auch bei allen geprüften Sensoren IMMER sensor.community als Quelle angibt).
